datasets/europarl_bilingual/README.md

```diff
@@ -0,0 +1,173 @@
+[Needs More Information]
```

Comment:
Could you add the dataset tags here please ?
You can find more information about that here:
https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card